src/test/java/com/epam/automation/enums/MainMenu.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.automation.enums;
+
+public enum MainMenu {
+    ;
+    public static final String WHAT_WE_DO_MENU = "/what-we-do";
```

Comment:
This approach for storing finals is ok. but personally I like to use text/config files for this.

Reply:
Should I use BufferReader?